src/discof/shredcap/fd_shredcap_tile.c

```diff
@@ -0,0 +1,445 @@
+#define _GNU_SOURCE  /* Enable GNU and POSIX extensions */
+#include "../../disco/tiles.h"
+#include "../../disco/topo/fd_topo.h"
+
+#include <errno.h>
+#include <fcntl.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#include <string.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <linux/unistd.h>
+#include <sys/socket.h>
+#include <linux/if_xdp.h>
+#include "generated/fd_shredcap_tile_seccomp.h"
+
+#include "../../disco/net/fd_net_tile.h"
+#include "../../flamenco/types/fd_types.h"
+
+/* This is starting to look pretty similar to the archiver writer/feeder,
```

Comment:
don't need to mention archiver writer / feeder concepts would instead just document that it essentially spies on the shred and repair_net links